### PR TITLE
Update Makefile for 1.12.2 release

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
 # Release Notes
 
+## Version 1.12.2 - 2020-08-03
+
+Features:
+* Automated GCP Credentials Addon Prototype [#8682](https://github.com/kubernetes/minikube/pull/8682)
```
Comment (medyagh) [inline CHANGELOG.md:6]: Change this to
New Addon: and remove prototype word

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+
+Bug Fixes:
+* docker/podman: warn if allocated memory is below limit [#8718](https://github.com/kubernetes/minikube/pull/8718)
+* Make sure to prepare the kic volume after creation [#8777](https://github.com/kubernetes/minikube/pull/8777)
+* Generate cri-o container runtime preload tarball [#8581](https://github.com/kubernetes/minikube/pull/8581)
+* Enabling metrics addon when someone enables dashboard [#8842](https://github.com/kubernetes/minikube/pull/8842)
+* make base-image respect --image-repository [#8880](https://github.com/kubernetes/minikube/pull/8880)
+* restart kubelet if config changes [#8817](https://github.com/kubernetes/minikube/pull/8817)
+* UI: suggest to enable `metric-server` for full feature dashboard addon. [#8863](https://github.com/kubernetes/minikube/pull/8863)
+* Add minimal minikube-automount to the kic image [#8780](https://github.com/kubernetes/minikube/pull/8780)
```
Comment (medyagh) [inline CHANGELOG.md:21]: Move this to bugs, and make it usable for the user, 
Fix mount issue in Docker/Podman Drivers

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+* Add stopping as a possible state in deleting, change errorf to warningf [#8896](https://github.com/kubernetes/minikube/pull/8896)
+* Use preloaded tarball for cri-o container runtime [#8588](https://github.com/kubernetes/minikube/pull/8588)
+
+Bug Fixes:
+* docker/podman: warn if allocated memory is below limit [#8718](https://github.com/kubernetes/minikube/pull/8718)
+* Make sure to prepare the kic volume after creation [#8777](https://github.com/kubernetes/minikube/pull/8777)
+* Generate cri-o container runtime preload tarball [#8581](https://github.com/kubernetes/minikube/pull/8581)
+* Enabling metrics addon when someone enables dashboard [#8842](https://github.com/kubernetes/minikube/pull/8842)
+* make base-image respect --image-repository [#8880](https://github.com/kubernetes/minikube/pull/8880)
+* restart kubelet if config changes [#8817](https://github.com/kubernetes/minikube/pull/8817)
```
Comment (medyagh) [inline CHANGELOG.md:19]: re-word this to a way user would recongize it... like what was the Error we fixed.
"Fix upgrading from old minkube versions"

Comment (sharifelgamal) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (sharifelgamal) [reply CHANGELOG.md]: that was already in bugs. fixed wording.

File: CHANGELOG.md (hunk 1 above)
Comment (sharifelgamal) [reply CHANGELOG.md]: done

**Pull request merged**